src/store/index.js

```diff
@@ -1,9 +1,13 @@
 import {createStore, applyMiddleware, compose, combineReducers} from "redux";
 import thunk from "redux-thunk";
-import simpleReducer from "./simpleReducer";
+import simpleReducer from "./simpleReducer/simpleReducer";
+import authorsReducer from "./authors/authorsReducer";
+import booksReducer from "./books/booksReducer";
 
 export const reducer = combineReducers({
     simpleReducer,
```

Comment:
let's remove simpleReducer and action since now we have real reducers and actions